### PR TITLE
Fix LuaAccessor when write=false.

### PR DESCRIPTION
**Pull request by franciscovalentecastro**

### CODE REVIEW

File: confgenerator/filter/internal/ast/ast.go [hunk 1]
```diff
@@ -148,7 +148,8 @@ then
 			fmt.Fprintf(&out, `record[%s] = {}
 `, p)
 		} else {
-			fmt.Fprintf(&out, `return nil`)
+			fmt.Fprintf(&out, `return nil
```
Comment (quentinmit) [inline confgenerator/filter/internal/ast/ast.go:151]: Nit: I think this would be slightly clearer as `"return nil\n"`

**Review by quentinmit — APPROVED** [review]

File: confgenerator/filter/internal/ast/ast.go (hunk 1 above)
Comment (franciscovalentecastro) [reply confgenerator/filter/internal/ast/ast.go]: Fixed.

**Review by quentinmit — APPROVED** [review]
New unit test LGTM

**Comment by franciscovalentecastro** [comment]
Third party apps integration test (Windows) are failing due to an unrelated issue.
Screen Shot 2022-04-29 at 15 30 35

**Pull request merged**